scripts/windows/SetDeploymentTokenScript.ps1

```diff
@@ -0,0 +1,35 @@
+$Token = "";
+
+if ([string]::IsNullOrEmpty($Token))
+{
+    Write-Host "Deployment token not provided. Exiting."
+    Exit
+}
+
+# get stored deployment token value
+$hklm = [Microsoft.Win32.RegistryKey]::OpenBaseKey([Microsoft.Win32.RegistryHive]::LocalMachine, [Microsoft.Win32.RegistryView]::Registry32)
+$key =  $hklm.OpenSubKey("SOFTWARE\Zorus Inc.\Archon Agent")
+$previousToken = $key.GetValue("deploymentKey")
+
+# get installation path
+$path = Get-ItemProperty -Path $zorusRegistry -Name Path 2>$null | Select-Object -ExpandProperty Path
```

Comment:
The install path needs to use the new registry thing too since `$zorusRegistry` is no longer a thing.

Reply:
yupp, im an idiot